_data/cloud.yml

```
@@ -43,13 +43,13 @@ websites:
       - totp
     doc: https://www.digitalocean.com/docs/accounts/security/2fa/
 
-  - name: EngineYard
+  - name: Engine Yard
     url: https://www.engineyard.com
     img: engineyard.png
     tfa:
       - totp
-    doc: https://blog.engineyard.com/2015/engine-yards-two-factor-authentication
-    exception: "To enable 2FA on EngineYard, contact support."
+    doc: https://blog.engineyard.com/engine-yards-two-factor-authentication
```

That doc link talks more about what 2FA is than how users should enable it. I think this link works better:
```suggestion
    doc: https://support.cloud.engineyard.com/hc/en-us/community/posts/203752678
```